pkg/defaults/defaults.go

```diff
@@ -189,6 +189,10 @@ const (
 	// DNSProxyLockCount.
 	DNSProxyLockTimeout = 500 * time.Millisecond
 
+	// DNSProxySocketLingerTimeout defines how long we wait for the connection
+	// between the DNS proxy and the upstream server to be closed.
+	DNSProxySocketLingerTimeout = 10
```

Comment:
nit: Might be good here to specify the unit (second) in the comment for posterity.